test/apm/resources/traceid_generator.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"crypto/rand"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"time"
+)
+
+func main() {
```

Comment:
I understand that this is probably to avoid dependencies, but https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/propagators/aws/xray provides an ID generator, which is what customers would be using as well.

Reply:
oh nice! I can look into this